[2022-02-17 13:52:18,270] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.task_a manual__2022-02-17T13:52:16.549448+00:00 [queued]>
[2022-02-17 13:52:18,281] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.task_a manual__2022-02-17T13:52:16.549448+00:00 [queued]>
[2022-02-17 13:52:18,281] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-17 13:52:18,282] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-17 13:52:18,282] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-17 13:52:18,296] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_a> on 2022-02-17 13:52:16.549448+00:00
[2022-02-17 13:52:18,300] {standard_task_runner.py:52} INFO - Started process 4787 to run task
[2022-02-17 13:52:18,311] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'task_a', 'manual__2022-02-17T13:52:16.549448+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpc1t6gv6r', '--error-file', '/tmp/tmp7kasn8rt']
[2022-02-17 13:52:18,313] {standard_task_runner.py:77} INFO - Job 35: Subtask task_a
[2022-02-17 13:52:18,389] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.task_a manual__2022-02-17T13:52:16.549448+00:00 [running]> on host f8b49ae6ae82
[2022-02-17 13:52:18,472] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=task_a
AIRFLOW_CTX_EXECUTION_DATE=2022-02-17T13:52:16.549448+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-17T13:52:16.549448+00:00
[2022-02-17 13:52:18,473] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-17 13:52:18,474] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls -a']
[2022-02-17 13:52:18,494] {subprocess.py:85} INFO - Output:
[2022-02-17 13:52:18,550] {subprocess.py:89} INFO - .
[2022-02-17 13:52:18,550] {subprocess.py:89} INFO - ..
[2022-02-17 13:52:18,551] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-17 13:52:18,583] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=task_a, execution_date=20220217T135216, start_date=20220217T135218, end_date=20220217T135218
[2022-02-17 13:52:18,616] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-17 13:52:18,662] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
